.storybook/cypress/integration/storybook_spec.js

```diff
@@ -1,17 +1,10 @@
-import { testResponseCode } from '../../../cypress/integration/test-helper';
-
 describe('Storybook Article', () => {
-  beforeEach(() => {
+  // eslint-disable-next-line no-undef
+  before(() => {
```

Comment:
If we are just doing a single page visit surely `before(() => {` isn't required and `cy.visit('/');` can just be the first line in the describe block?

Reply:
Tried this locally `cy.visit` requires being within a running test. EG it(), before(), beforeEach()